src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppresswithnearbytextfilter/InputSuppressWithNearbyTextFilterVariableNearbyTextPatternAndInfluenceFormat.xml

```diff
@@ -0,0 +1,28 @@
+/*
+SuppressWithNearbyTextFilter
+nearbyTextPattern = SUPPRESS CHECKSTYLE (\\w+) - ([+-]\\d+) Lines
+checkPattern = $1
+messagePattern = (default)(null)
+idPattern = (default)(null)
+influenceFormat = $2
```

Comment:
...Pattern
...Pattern
...
...Format

influenceFormat is a bit weird. It is number and it is string for regexp. Should we name it just "influence" and explain in doc that it can have some interesting values.

Reply:
Just `influence` is better. Or even `rowInfluence`?

Reply:
Done. Renamed to `rowInfluence`.

Reply:
renamed to lineRange